### PR TITLE
Update labels and descriptions

### LINKED ISSUE #409: Add `triage` label
### Subject

GitHub

### Description

Currently, when a bug fix issue is created, no label is assigned.
A `triage` label will be added when created and prioritized.

### Are you willing to participate in the development of this feature and create a pull request?

None

### LINKED ISSUE #410: There is an error in the issue template.
### Provide the exact quote of the error or issue

Issue template was imported from Yamada Colors and needs to be changed.

### Does the feature already exist? Please link to it.

_No response_

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #410
Closes #409

## Description

Update labels and descriptions.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.ja.yml [hunk 1]
```diff
@@ -21,8 +22,8 @@ body:
       required: true
   - type: input
     attributes:
-      label: "機能はすでに存在しますか？リンクを貼ってください。"
-      description: "例: https://yamada-components.app/colors/ffffff"
+      label: "コンポーネントはすでに存在しますか？リンクを貼ってください。"
+      description: "例: https://yamada-components.app/layouts/headers/simple-header"
   - type: dropdown
```
Comment (odendayoko) [inline .github/ISSUE_TEMPLATE/bug_report.ja.yml:27]: @hirotomoyamada 
The link https://yamada-components.app/layouts/headers/simple-header appears to be invalid. 
No Problem?

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
 body:
   - type: markdown
     attributes:
       value: |
-        Thank you for submitting a feature request! 😎
+        Thank you for submitting a component request! 😎
 
-        If you have an idea for a new feature topic, notice something is not working as intended, or feel there is an error with the current functionality, you are in the right place!
+        If you have an idea for a new component or notice something is not working as intended, you are in the right place!
   - type: input
```
Comment (odendayoko) [inline .github/ISSUE_TEMPLATE/feature_request.yml:11]: @hirotomoyamada 
The English version removes `, or feel there is an error with the current functionality`, while the Japanese version keeps the corresponding phrase `または現在の機能に何か誤りがあると感じた場合は、`.

I suggest verifying which modification is intended and ensuring consistency between both language versions.

**Review by odendayoko — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.ja.yml (hunk 1 above)
Comment (hirotomoyamada) [reply .github/ISSUE_TEMPLATE/bug_report.ja.yml:27]: @odendayoko 

This is not a problem.
It is only a sample.

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (hirotomoyamada) [reply .github/ISSUE_TEMPLATE/feature_request.yml:11]: Fixed.

File: .github/ISSUE_TEMPLATE/feature_request.ja.yml [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
-        新しい機能のトピックについてのアイデアがある、何かが適切に機能されていないと気づいた、または現在の機能に何か誤りがあると感じた場合は、ここが正解です！
+        新しいコンポーネントについてのアイデアがある場合は、ここが正解です！
   - type: "input"
     id: "subject"
     attributes:
       label: "主題"
-      description: "この機能の主題は何ですか？"
-      placeholder: "コントラスト"
+      description: "このコンポーネントの主題は何ですか？"
+      placeholder: "Header"
```
Comment (cidkumagai) [inline .github/ISSUE_TEMPLATE/feature_request.ja.yml:16]: Is it okay to leave this placeholder in English?

**Review by odendayoko — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.ja.yml (hunk 3 above)
Comment (hirotomoyamada) [reply .github/ISSUE_TEMPLATE/feature_request.ja.yml:16]: No problem.Component names are always in English.

**Review by cidkumagai — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
@odendayoko @odendayoko 

Please resolve the review conversation.
Otherwise, you will not be able to merge.

**Pull request merged**